### ISSUE TITLE
Request: support direct navigation to deep-link items in Chromium

### ISSUE THREAD

**Issue by cxw42** [semver/semver.org#241]
Tested on Chromium 77 x64, Ubuntu 18.04.2 LTS.

Expected: When I open a deep link to a specification paragraph from outside semver.org, I will be taken to that paragraph.

Observed: When I open <https://semver.org/spec/v2.0.0.html#spec-item-9>, I am left at the top of the page and not taken directly to the numbered paragraph of the specification.

However:
- If I go to <https://semver.org/spec/v2.0.0.html> and click on any of the deep link icons, I am taken directly to the paragraph in question.  
- If I go to <https://semver.org/spec/v2.0.0.html#backusnaur-form-grammar-for-valid-semver-versions>, I am taken to the BNF section (#128).

I see that `anchorli.js` adds IDs (see #129):

https://github.com/semver/semver.org/blob/7bc8e4d9800601ec46155704d243965fbf22a1a9/js/anchorli.js#L11-L18

Could it be updated to also add anchors, or to check for a fragment in the requested URL on page load, and jump there?

Thank you for considering this request!

**Comment by klehelley** [comment]
I've tested with Firefox 69.0.1 and it works there in all my tests. However, I have observed that with Chrome 64 it only works from time to time.

Personally I tend to question the use of dynamic content in such a simple page. Using Javascript to add the anchors seems to me to be more work than just having added the anchors manually; but I guess that is related to the way the page is generated? I haven't used Jekyll in years.
